src/paths/widest_paths.c

```diff
@@ -0,0 +1,747 @@
+ *        and fill the IDs of the vertices along the paths from/to
+ *        the vertices. Supply a null pointer here if you don't need
+ *        these vectors. Normally, either this argument, or the \c
+ *        vertices should be non-null, but no error or warning is given
+ *        if they are both null pointers.
+ * \param from The id of the vertex from/to which the widest paths are
+ *        calculated.
+ * \param to Vertex sequence with the IDs of the vertices to/from which the
+ *        widest paths will be calculated. A vertex might be given multiple
+ *        times.
+ * \param weights The edge weights. Edge weights can be negative. If this
+ *        is a null pointer or if any edge weight is NaN, then an error
+ *        is returned.
+ * \param mode The type of widest paths to be use for the
+ *        calculation in directed graphs. Possible values:
+ *        \clist
+ *        \cli IGRAPH_OUT
+ *          the outgoing paths are calculated.
+ *        \cli IGRAPH_IN
+ *          the incoming paths are calculated.
+ *        \cli IGRAPH_ALL
+ *          the directed graph is considered as an
+ *          undirected one for the computation.
+ *        \endclist
+ * \param predecessors A pointer to an initialized igraph vector or null.
+ *        If not null, a vector containing the predecessor of each vertex in
+ *        the single source widest path tree is returned here. The
+ *        predecessor of vertex i in the tree is the vertex from which vertex i
+ *        was reached. The predecessor of the start vertex (in the \c from
+ *        argument) is itself by definition. If the predecessor is -1, it means
+ *        that the given vertex was not reached from the source during the
+ *        search. Note that the search terminates if all the vertices in
+ *        \c to are reached.
+ * \param inbound_edges A pointer to an initialized igraph vector or null.
+ *        If not null, a vector containing the inbound edge of each vertex in
+ *        the single source widest path tree is returned here. The
+ *        inbound edge of vertex i in the tree is the edge via which vertex i
+ *        was reached. The start vertex and vertices that were not reached
+ *        during the search will have -1 in the corresponding entry of the
+ *        vector. Note that the search terminates if all the vertices in
+ *        \c to are reached.
+ * \return Error code:
+ *        \clist
+ *        \cli IGRAPH_ENOMEM
+ *           not enough memory for temporary data.
+ *        \cli IGRAPH_EINVVID
+ *           \p from is invalid vertex ID, or the length of \p to is
+ *           not the same as the length of \p vertices (if not NULL) or
+ *           the length of \p edges (if not NULL)
+ *        \cli IGRAPH_EINVMODE
+ *           invalid mode argument.
+ *        \endclist
+ *
+ * Time complexity: O(|E|log|E|+|V|), where |V| is the number of
+ * vertices and |E| is the number of edges
```

Comment:
Vertices and edges of the graph? Or vertices in the `to`?

Reply:
|V| is the number of nodes in the graph, since even if there was only one node in `to`, the Dijkstra could explore every other node in the graph first if that single node is the furthest from the source. 

I'll update the comment to clarify this.